### ISSUE TITLE
Seek for help: about model training and its structure

### ISSUE THREAD

**Issue by BlackRab** [ph-w2000/DuoCast#2]
Hi. This is a nice work! Recently, I wanted to conduct a simple test, so I trained the model using approximately 1200 samples (my own data) and set the epoch to 20. When I tested the model, I found the results of the last 5 frames to be quite strange; there was no clear temporal continuity between the last 5 frames and the first 15 frames. This is likely related to the small number of training samples and epochs. I would like to ask how many epochs are generally appropriate for training, and what the approximate sample size should be? Also, what is a recommended learning rate?

Additionally, I've noticed that the current model only works for single-channel data. Therefore, I'd like to know if the existing model will be extended to support multi-channel data? Or, if I wish to extend the model to accommodate multiple channels, do you have any suggestions?

I look forward to your reply, thank you!

**Comment by ph-w2000** [comment]
Hi @BlackRab . Thank you for your interest in our work. As for the last 5 frames issue, there could several reasons: 1. the high-freq model is not well-trained (you may train with more data or the VAE is not well-trained) 2. the diffusion steps for the high-freq model could be adjusted. Our model was trained with 50 epochs on SEVIR dataset with input size 128x128. 

currently, we dont plan to extend the model to multi-channel data. you may try adjusting the input channel number to adapt the model to work with multi-channel input

**Comment by BlackRab** [comment]
Thanks for your reply! I will train the model with more data and larger epochs, and then observe whether the results improve.
